src/components/Filter/Filter.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './Filter.css'
+
+class Filter extends Component {
+    constructor(props){
+        super(props);
+        this.handleFilterChange = this.handleFilterChange.bind(this);
+    }
+
+    handleFilterChange = (event) => {
+        this.props.onSelectFilter(this.props.type, event.target.value);
+    }
+
+    removePrefix(text){
+        var arr = text.split("_");
+        arr.splice(0,1);
+        var string = arr.join(" ");
+        return string.charAt(0).toUpperCase() + string.slice(1)
+    }
+
+    render() {
+        const options = this.props.options;
+
+        return (
+            <div>
+                <h1 className="filter-name">{this.removePrefix(this.props.type)}</h1>
+                <select className="filter-select" onChange={this.handleFilterChange} value={this.value}>
+                    <option value="">I don't care</option>
+                    {options.map(op => {
+                        return <option value={op}>{op}</option>
```

Comment:
When you `map` into a bunch of React nodes like this, always add a `key` attribute: https://reactjs.org/docs/lists-and-keys.html

It's also recommended to set the key to be a value (not sure if has to be a string) that _doesn't depend on the item's index_.
So something like this is not recommended:

```jsx
options.map((op, index) => {
  return <option key={`option-${index}`} value={op}>{op}</option>;
});
```

But this is OK:
```jsx
options.map(op => {
  return <option key={`option-${op}`} value={op}>{op}</option>;
});
```